修正1

Takahashihiroak opened the pull request:
商品のデータがない場合は、ダミー商品が表示されている動画https://gyazo.com/2048c8ebe20001d5d68fb3d845798bb2
商品のデータがある場合は、商品が一覧で表示されている動画
https://gyazo.com/c6d7a69895415348a234fbc40fc17026

Takahashihiroak committed:
修正1

Akinori0123 commented on app/controllers/items_controller.rb:
```diff
@@ -2,6 +2,7 @@ class ItemsController < ApplicationController
   before_action :authenticate_user!, only: [:new]
 
   def index
+    @items = Item.all
```
```
@items = Item.all
```
トップページで商品が出品された順になるよう、@itemsの定義において並び替え処理を行いましょう。

Akinori0123 commented on app/views/items/index.html.erb:
```diff
@@ -127,57 +127,62 @@
     </div>
     <ul class='item-lists'>
+        <% @items.each do |item| %>
+          <%# 商品のインスタンス変数になにか入っている場合、中身のすべてを展開できるようにしましょう %>
```
```
<%# 商品のインスタンス変数になにか入っている場合、中身のすべてを展開できるようにしましょう %>
```
実装案内コメントにつきましては実装完了後削除しましょう。
企業提出時に実装案内コメントが残っていることは好ましくないためです。
（161行目付近にある閉じのコメントも同様に削除していただければと思います）

同ファイルにある他の実装済みの実装案内コメントも削除しましょう。

Akinori0123 reviewed (commented):
レビュー依頼いただきありがとうございます。コミット内容確認いたしました。
二点程コメントさせていただきましたので、ご確認の程よろしくお願いいたします。

また、他者に見せることを想定してプルリクエストには以下を毎回記載していただくとより良くなります。
・何の実装かを明確に示したプルリクエストのタイトル（現在「修正1」となっているものです）
・WhatとWhy
両者ともプルリクエストのトップページから編集できますので、こちらも修正の程よろしくお願いいたします。

Takahashihiroak committed:
商品一覧表示機能

また、他者に見せることを想定してプルリクエストには以下を毎回記載していただくとより良くなります。
・何の実装かを明確に示したプルリクエストのタイトル（現在「修正1」となっているものです）
・WhatとWhy
両者ともプルリクエストのトップページから編集できますので、、

こちらのコメントに関してどのような表記が正しいのでしょうか？

tsuyoshimizu reviewed (commented):
修正とレビュー依頼ありがとうございます。

修正点確認いたしました。
実装できていることが確認できましたので、LGTMとさせていただきます。
実装お疲れ様でした。
Whatとwhyの書き方については次回からお願いいたします。
また、ご不明点ございましたら、コネクトにてご質問お願いいたします。

こちらのブランチは忘れずにMergeを行っていただき、ローカルのmasterブランチにPullしましょう。
その後、新しいブランチを作成し次の実装に取り組んでいただけますと幸いです。
ここまでの実装、本当にお疲れ様でした。
引き続き、応援しております。
